bwapi/BWAPI/Source/BWAPI/GameInternals.cpp

```diff
@@ -324,9 +324,6 @@ namespace BWAPI
     flags.fill(false);
 
     // Clear the latency buffer
-    for(unsigned int j = 0; j < this->commandBuffer.size(); ++j)

```

Comment:
Is there code that clears the latency compensation buffer between games?

Reply:
As you can see at
https://github.com/bwapi/bwapi/pull/783/files#diff-6464d0d2ecc3b9271d1d8112b1a20331L319,
the vector now contains value types. The `clear()` at https://github.com/bwapi/bwapi/pull/783/files#diff-5b9870b36b881f8a0e19b037bcb74c17L330 then does everything. There are no more pointers in the latcom code.